lib/rubocop/cop/mixin/code_length.rb

```diff
@@ -24,9 +24,19 @@ def check_code_length(node, *_)
         end
       end
 
+      def ignore_list
+        cop_config['Ignore'] || []
+      end
+
+      def ignore_line?(source_line)
+        ignore_list.find { |o| source_line.include?(o) }
```

Comment:
I'm not sure this is a good approach:

- it will yield some positives
- it will fail for multi-line method calls

Something more robust would involve checking for a specific method receiver or methods at the beginning of the line or something like this.